gensim/matutils.py

```diff
@@ -533,7 +533,17 @@ def jaccard(vec1, vec2):
 
 
 def jaccard_set(set1, set2):
-    return 1. - float(len(set1 & set2)) / float(len(set1 | set2))
+    """
+    A distance metric between set representation.
+    Returns 1 minus the intersection divided by union.
```

Comment:
PEP257: docstrings in imperative mode ("do X, return Y", not "does X, returns Y").